### PR TITLE
Add note about potential lockouts when configuring a policy

### PR DESCRIPTION
**Pull request by hslatman**

### CODE REVIEW

File: src/pages/docs/step-ca/policies.mdx [hunk 1]
```diff
@@ -284,6 +284,16 @@ Allow all DNS subdomains of `"local"` in SSH host certificates on authority leve
 
 You can find more examples in the `step` CLI command reference for the [`step ca policy`](https://smallstep.com/docs/step-cli/reference/ca/policy) namespace.
 
+<Alert severity="info">
+  <AlertTitle>Policies and Remote Management authentication</AlertTitle>
+  <div>
+    The authentication scheme for Remote Management uses short-lived X.509 certificates to authenticate (super) admin users.
```
Comment (tashian) [inline src/pages/docs/step-ca/policies.mdx]:
```suggestion
    The authentication scheme for Remote Management uses short-lived X.509 certificates to authenticate Admin and Super Admin users.
```

File: src/pages/docs/step-ca/policies.mdx [hunk 2]
```diff
@@ -284,6 +284,16 @@ Allow all DNS subdomains of `"local"` in SSH host certificates on authority leve
...
+    When an admin requests access when an X.509 policy is active, the policy must allow the subject of the admin, or else the admin won't be able to sign in.
```
Comment (tashian) [inline src/pages/docs/step-ca/policies.mdx]:
```suggestion
    When an Admin requests access when an X.509 policy is active, the policy must allow the subject of the Admin, or else the Admin won't be able to sign in.
```

File: src/pages/docs/step-ca/policies.mdx [hunk 3]
```diff
@@ -284,6 +284,16 @@ Allow all DNS subdomains of `"local"` in SSH host certificates on authority leve
...
+    To protect users from locking themselves out, a new or updated policy will be checked to not lock out users, indicating issues to the user, resulting in an error like <Code>the provided policy would lock out [step] from the CA</Code>.
+    The policy can then be adjusted to include the (super) admin subject(s), e.g. <Code>step ca policy authority x509 allow dns step</Code>.
```
Comment (tashian) [inline src/pages/docs/step-ca/policies.mdx]:
```suggestion
    The policy can then be adjusted to include the Admin subject(s), e.g. <Code>step ca policy authority x509 allow dns step</Code>.
```

**Review by tashian — CHANGES_REQUESTED** [review]
I've just changed the capitalization to match what's on the Remote Management section of the Provisioners doc. Otherwise lgtm!

**Comment by hslatman** [comment]
Committed your suggestions. Thanks! 🙂

**Review by tashian — APPROVED** [review]

**Pull request merged**